### PR TITLE
Move "Hardware sizing guidelines" to Reference section

### PR DESCRIPTION
**Pull request by pierwill**
Seems to me this makes more sense here.

- [x] Tests pass (no build errors)
- [x] Rebased/mergeable

### CODE REVIEW

**Comment by pierwill** [comment]
### TODO
- [ ] update links

File: content/enterprise_influxdb/v1.9/guides/hardware_sizing.md [hunk 1]
```diff
@@ -3,9 +3,7 @@ title: Hardware sizing guidelines
 Description: >
   Review configuration and hardware guidelines for InfluxDB OSS (open source) and InfluxDB Enterprise.
 menu:
-  enterprise_influxdb_1_9:
-    weight: 40
-    parent: Guides
+  enterprise_influxdb_1_9_ref
```
Comment (sanderson) [inline content/enterprise_influxdb/v1.9/guides/hardware_sizing.md:6]: This still needs a weight.

**Review by sanderson — CHANGES_REQUESTED** [review]
Where you just changed the menu key, you don't actually have to update any links. The filepaths stay the same. For the 1.x docs, I think this is the way to go.

**Comment by pierwill** [comment]
As in, links to `/enterprise_influxdb/v1.9/guides/hardware_sizing/` would be redirected to `/enterprise_influxdb/v1.9/reference/hardware_sizing/`? Still, it's a bit confusing to future editors if we don't update the links, IMO.

**Comment by pierwill** [comment]
Ah, I see. I think it's best to move the file as well.

File: content/enterprise_influxdb/v1.9/reference/hardware_sizing.md [hunk 2]
```diff
@@ -3,9 +3,8 @@ title: Hardware sizing guidelines
 Description: >
   Review configuration and hardware guidelines for InfluxDB OSS (open source) and InfluxDB Enterprise.
 menu:
-  enterprise_influxdb_1_9:
+  enterprise_influxdb_1_9_ref:
     weight: 40
```
Comment (sanderson) [inline content/enterprise_influxdb/v1.9/reference/hardware_sizing.md:7]:
```suggestion
menu: enterprise_influxdb_1_9_ref
weight: 40
alias:
  - /enterprise_influxdb/v1.9/guides/hardware_sizing/
```

File: content/enterprise_influxdb/v1.9/reference/hardware_sizing.md [hunk 3]
```diff
@@ -3,9 +3,8 @@ title: Hardware sizing guidelines
 Description: >
   Review configuration and hardware guidelines for InfluxDB OSS (open source) and InfluxDB Enterprise.
 menu:
```
Comment (sanderson) [inline content/enterprise_influxdb/v1.9/reference/hardware_sizing.md:5]: If you accept my suggestion below, remove this line.

**Review by sanderson — CHANGES_REQUESTED** [review]

**Review by sanderson — APPROVED** [review]

**Pull request merged**